website/src/components/product/GitLabIntegrationSection.tsx

```diff
@@ -0,0 +1,39 @@
+import ArrowRightBoxIcon from 'mdi-react/ArrowRightBoxIcon'
+import * as React from 'react'
+import { RequestDemoAction } from '../../css/components/actions/RequestDemoAction'
+
+export const GitLabIntegrationSection: React.FunctionComponent<{ className?: string }> = ({
+    className = '',
+}) => (
+
+    <div className="gitlab-integration-section py-5 row justify-content-center ">
+        <div className="row justify-content-center container">
+            <h2 className="text-center display-4 mb-2">
+                GitLab adds code navigation with Sourcegraph
```

Comment:
Some kind of "New" badge would be good here - like:
![image](https://user-images.githubusercontent.com/1976/68718767-17599400-055f-11ea-8a31-1b9f258eadaf.png)

Reply:
@dadlerj The styles for the `private beta` on the automation page may help.

Reply:
Yep that's exactly what I pulled from!